app/src/main/java/ch/hikemate/app/ui/components/HikeCard.kt

```diff
@@ -99,11 +100,15 @@ fun HikeCard(
                     elevationData = elevationData,
                     modifier =
                         Modifier.weight(1f)
+                            .zIndex(1.0f)
```

Comment:
Question: Why did you add this ? After removing it, nothing changes in my side.

Reply:
Oh yeah, just forgot it during testing, thanks!